src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,262 @@
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-05-03',
+        },
+        {
+          project: 'Servo',
+          description:
+            '1446768 - Only post "No taskcluster jobs.." to a PR once',
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-05-01',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1457126 - Authorization failures should state which clientId lacks scopes',
+          tag: 'Rust',
+          assignedTo: 'None',
+          lastUpdated: '2018-04-05',
+        },
+        {
+          project: 'Taskcluster',
+          description: '44 - Add pagination to auth.listRoles',
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-03-05',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1306494 - Add a diff+commit submit button to some text areas in tc-tools',
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-02-12',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '333 - Make azure-entities and azure-blob0-storage independent of tc-lib-monitor',
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-02-04',
+        },
+        {
+          project: 'Taskcluster',
+          description: '1344912 - Support tag events, too',
+          tag: 'JS',
+          assignedTo: 'None',
+          lastUpdated: '2018-01-01',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1453714 - Return http 424 instead of 403 for error artifacts',
+          tag: 'Python',
+          assignedTo: 'None',
+          lastUpdated: '2017-12-04',
+        },
+      ].sort((a, b) => (a.lastUpdated > b.lastUpdated ? -1 : 1)),
+      page: 0,
+      rowsPerPage: PROJECTS_PAGE_SIZE,
+    };
+  }
+
+  createSortHandler = property => event => {
+    this.handleRequestSort(event, property);
+  };
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    const data =
+      order === 'desc'
+        ? this.state.data.sort((a, b) => (b[orderBy] < a[orderBy] ? -1 : 1))
+        : this.state.data.sort((a, b) => (a[orderBy] < b[orderBy] ? -1 : 1));
+
+    this.setState({ data, order, orderBy });
+  };
+
+  handlePageChange = (event, page) => {
+    this.setState({ page });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { data, order, orderBy, rowsPerPage, page } = this.state;
+
+    return (
+      <div className={classes.root}>
+        <Toolbar>
+          <Typography variant="title" id="tableTitle">
+            Bugs and Issue
+          </Typography>
+          <div className={classes.spacer} />
+        </Toolbar>
```

Comment:
Remove the title from here. See my other comment about where to put it. Also, can we make it "Bugs & Issues" :)